api/cpp/CMakeLists.txt

```diff
@@ -119,6 +119,7 @@ define_cargo_dependent_feature(accessibility "Enable integration with operating
 define_cargo_dependent_feature(testing "Enable support for testing API (experimental)" ON "NOT SLINT_FEATURE_FREESTANDING")
 define_cargo_feature(experimental "Enable experimental features. (No backward compatibility guarantees)" OFF)
 define_cargo_dependent_feature(system-testing "Enable system testing support (experimental)" OFF "SLINT_FEATURE_EXPERIMENTAL AND NOT SLINT_FEATURE_FREESTANDING")
+define_cargo_feature(raw-window-handle-06 "Enable integration with raw-window-handle." OFF)
```

Comment:
The reason it is a feature in Rust is because it is dependent of a specific Rust crate (raw-window-handle) and a specific version (0.6) and that in the future, Rust users may opt for a newer version of raw-window-handle and so wouldn't enable this feature anymore.

In C++ we don't have this as we'd have to re-expose the API anyway, so IMHO it shouldn't need to be a feature and it can be enabled by default